popup.js

```diff
@@ -0,0 +1,179 @@
+    title: 'Profesional Art Printing Data',
+    apps: ['html', 'Bootstrap', 'Ruby on Rails'],
+    image: '/images/second-place.png',
+    desktopImage: '/images/placeholder1.png',
+    p: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    btnText: '#',
+    btnText1: '#',
+  },
+
+  {
+    item: 6,
+    title: 'Profesional Art Printing Data',
+    apps: ['html', 'Bootstrap', 'Ruby on Rails'],
+    image: '/images/second-place.png',
+    desktopImage: '/images/placeholder2.png',
+    p: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    btnText: '#',
+    btnText1: '#',
+  },
+];
+divHead.appendChild(divTitle);
+let i = 0;
+works.forEach((element) => {
+  i += 1;
+  const ulElement = document.createElement('ul');
+  const nombre = `ul-list${String(i)}`;
+  ulElement.id = 'ul-list';
+  ulElement.className = nombre;
+  ulElement.className = 'ul-list';
+  element.apps.forEach((app, iddex) => {
+    const liElement = document.createElement('li');
+    liElement.classList.add('tags-popup');
+    liElement.textContent = element.apps[iddex];
+    ulElement.appendChild(liElement);
+  });
+  divHead.appendChild(ulElement);
+});
+let posicionActualDelDesplazamiento;
+btnPopup.forEach((button, id) => {
+  button.addEventListener('click', () => {
+    const nId = works.findIndex((work) => work.item === id);
+    body.appendChild(windowShadow);
+    // body.style.mixBlendMode = 'multiply';
+    body.appendChild(popUp);
+    popUp.appendChild(divHead);
+    const numList = document.querySelectorAll('.ul-list');
+    numList.forEach((list, idd) => {
+      if (idd === id) {
+        list.style.display = 'flex';
+        list.classList.add('apps-popup');
+      } else {
+        list.style.display = 'none';
+      }
+    });
+    posicionActualDelDesplazamiento = window.scrollY;
+    // document.documentElement.scrollTop = 0;
+    // document.body.style.overflow = "hidden";
+    divTitle.appendChild(title);
+    divTitle.appendChild(closeBtn);
+    popUp.appendChild(divMiddle);
+    divMiddle.appendChild(imagePortrait);
+    divMiddle.appendChild(divFooter);
+    divFooter.appendChild(pLoret);
+    divFooter.appendChild(buttonContainer);
+    buttonContainer.appendChild(btnLive);
+    buttonContainer.appendChild(btnSee);
+    const windowScreen = document.querySelector('.window');
+    windowScreen.style.display = 'flex';
+    popUp.style.display = 'flex';
```

Comment:
- [ ] Please implement the close (x) button so that when the user clicks or taps it, the popup disappears 👍 
- [ ] Please ensure that the image of the project is showing on the left of the pop-up
![image](https://user-images.githubusercontent.com/46839548/216456883-c60256ea-92f7-404e-ae59-f72039b15263.png)

Reply:
hi, @codepantha I was checking the problem with the images and they work right. Check the attached video, please.
https://www.loom.com/share/602c5be958aa47f2acdda75d9aeb4ce8

Reply:
hi @codepantha , I deployed my popup branch and I could see the problem, sorry.

Reply:
No problem. Happy coding 🚀